acceptance/acceptance_test.go

```diff
@@ -2181,19 +2185,16 @@ func testAcceptance(
 								imageManager.CleanupImages(runImageName)
 							})
 
-							it("fails with a message", func() {
+							it("succeeds with a warning", func() {
```

Comment:
@joeybrown 

This test is failing when running the acceptance tests because in the previous `pack` release we were expecting the error message.

```bash
        --- FAIL: TestAcceptance/acceptance_suite/p_previous_cb_current_lc_current/stack_is_created/builder_is_created/build/builder_is_trusted_(and_set_as_default)/--run-image/the_run_image_has_the_wrong_stack_ID/succeeds_with_a_warning (3.02s)
```

Usually, when a new behavior,  we add a [Support](https://github.com/buildpacks/pack/blob/main/acceptance/invoke/pack.go#L200-L225) flag and use [SkipIf](https://github.com/buildpacks/pack/blob/main/acceptance/acceptance_test.go#L1099-L1101) with previous pack versions to avoid running that test.

In this case, maybe we can do the following:
- Add a new Support flag, valid from versions greater or equal to `0.37.0`
- Keep the current test, skip if the version is greater or equal to `0.37.0`
- Add the new test and skip it if the version is less than 0.37.0

Reply:
sounds great can do!